app/modules/node.c

```diff
@@ -448,6 +449,43 @@ static int node_compile( lua_State* L )
   return 0;
 }
 
+// Task callback handler for node.task.post()
+static task_handle_t do_node_task_handle;
+static void do_node_task (task_param_t task_fn_ref, uint8_t prio)
+{
+  UNUSED(prio);
```

Comment:
Bugger, it wasn't when I first wrote it and its just to avoid a compiler warning. I will be at my dev laptop sometime tomorrow.